app/views/cancelCase/CancellationReasonView.scala.html

```diff
@@ -38,7 +38,8 @@
             messages("cancellationReason.pageTitle"),
             true,
             true,
-            classes = Some("govuk-fieldset__legend")
+            classes = Some("govuk-fieldset__legend"),
+            maxCharacters = Some(1400)
```

Comment:
To maintain consistency, should we retrieve this value from CancellationReasonFormProvider.maxLength?